openslides/motions/static/templates/motions/motion-list.html

```diff
@@ -32,11 +32,27 @@ <h1 translate>Motions</h1>
 <table class="table table-striped table-bordered table-hover">
   <thead>
     <tr>
-      <th ng-click="toggleSort('identifier')" class="sortable">
+      <!-- TODO: Add motion.agenda_item to rest api -->
```

Comment:
@normanjaeckel I still missing motion.agenda_item in the rest api. Can you add this reference please?